packages/core/src/docs/data-model/text-x/build-utils/parse.ts

```diff
@@ -28,8 +28,6 @@ const tags = [
     DataStreamTreeTokenType.TABLE_CELL_END, // 表格开始
```

Comment:
Are some token types deprecated? If so, you should decorate `@deprecated` on them.